Cisco/cisco-secure-firewall/tests/test_ASA_313005.json

```diff
@@ -0,0 +1,68 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco Secure Firewall",
+        "dialect_uuid": "466aeca2-e112-4ccc-a109-c6d85b91bbcf"
+      }
+    },
+    "message": "%ASA-4-313005: No matching connection for ICMP error message: icmp src outside:1.2.3.4(LOCAL\\a.smithee) dst inside:1.4.3.2 (type 3, code 3) on outside interface.  Original IP payload: udp src 1.4.3.2/53 dst 1.2.3.4/60717."
+  },
+  "expected": {
+    "message": "%ASA-4-313005: No matching connection for ICMP error message: icmp src outside:1.2.3.4(LOCAL\\a.smithee) dst inside:1.4.3.2 (type 3, code 3) on outside interface.  Original IP payload: udp src 1.4.3.2/53 dst 1.2.3.4/60717.",
+    "event": {
+      "code": "313005",
+      "kind": "event",
+      "category": [
+        "network"
+      ]
+    },
+    "observer": {
+      "vendor": "Cisco",
+      "ingress": {
+        "interface": {
+          "alias": "outside"
+        }
+      },
+      "egress": {
+        "interface": {
+          "alias": "inside"
+        }
+      },
+      "product": "Adaptive Security Appliance"
+    },
+    "action": {
+      "target": "network-traffic",
+      "name": "no matching connection"
+    },
+    "cisco": {
+      "ftd": {
+        "icmp_code": "3",
+        "icmp_type": "3"
+      }
+    },
+    "destination": {
+      "ip": "1.4.3.2",
+      "address": "1.4.3.2"
+    },
+    "network": {
+      "transport": "icmp"
+    },
+    "source": {
+      "ip": "1.2.3.4",
+      "address": "1.2.3.4"
+    },
+    "user": {
+      "name": "a.smithee"
```

Comment:
Maybe you can extract the domain of the user (`Local`) and put it in `user.domain`?